lib/pure/collections/tables.nim

```diff
@@ -989,9 +989,10 @@ proc inc*[A](t: var CountTable[A], key: A, val = 1) =
 proc smallest*[A](t: CountTable[A]): tuple[key: A, val: int] =
   ## returns the (key,val)-pair with the smallest `val`. Efficiency: O(n)
   assert t.len > 0
-  var minIdx = 0
+  var minIdx = -1
   for h in 1..high(t.data):
```

Comment:
i think after your correction you need to start the loop at index 0.

Reply:
You're right